letta/schemas/agents_tags.py

```diff
@@ -1,33 +0,0 @@
-from datetime import datetime
```

Comment:
what was this replaced by?

Reply:
we no longer need the Pydantic object

Reply:
the table is purely relational now and needs no manager, so never need to convert to pydantic obj